logging/context.go

```diff
@@ -0,0 +1,55 @@
+package logging
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	internalLogLevel = map[LogLevel]zapcore.Level{ //nolint:gochecknoglobals
+		Debug: zapcore.DebugLevel,
+		Info:  zapcore.InfoLevel,
+		Warn:  zapcore.WarnLevel,
+		Error: zapcore.ErrorLevel,
+	}
+
+	locator = "stub" //nolint:gochecknoglobals
+
+	logLvl = zap.NewAtomicLevelAt(zap.DebugLevel) //nolint:gochecknoglobals
+
+	highPriority = zap.LevelEnablerFunc(func(lvl zapcore.Level) bool { //nolint:gochecknoglobals
+		return lvl >= zapcore.ErrorLevel && logLvl.Enabled(lvl)
+	})
+
+	lowPriority = zap.LevelEnablerFunc(func(lvl zapcore.Level) bool { //nolint:gochecknoglobals
+		return lvl < zapcore.ErrorLevel && logLvl.Enabled(lvl)
+	})
+
+	consoleEncoder zapcore.Encoder //nolint:gochecknoglobals
+	jsonEncoder    zapcore.Encoder //nolint:gochecknoglobals
```

Comment:
@stepanvalyavskiy to avoid so many global variables here, we can define internal private `struct`ure for base settings and assign it in `Init()` call:
```go
mux sync.Mutex

func Init(node Node) {
  mux.Lock()
  defer mux.Unlock()
  if log != nil {
    panic(errLogAlreadyInitialized)
  }
  log = &log{/*all params here*/}
}
```